thunder.install

```diff
@@ -29,7 +30,24 @@ function thunder_update_dependencies() {
     8006 => $installThunderUpdaterHook,
   ];
 
-  return $dependencies;
+  // Ensure that all thunder updates come after all the other update hooks.
+  $list = [];
+  $return = [];
+  $updates = update_get_update_list();
+  $module_data = system_rebuild_module_data();
+  $thunder_path_regex = '@^' . $module_data['thunder']->getPath() . '@';
```

Comment:
One question, when we use `drush make` build then all modules are in profile `modules` directory too. Fe. `.../thunder/profiles/thunder/modules`. Could that be problem?

Reply:
Totally will be a problem - nice spot. I guess a hardcoded list - I thought about matching on thunder_ but this feels icky.

Reply:
Latest commit has changed this to a hard-coded list.